src/Components/HouseholdDataBlock/HouseholdDataBlock.js

```diff
@@ -223,6 +223,8 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
 
   const createQuestionHeader = (personIndex) => {
     let header;
+    const headOfHHInfoWasEntered = formData.householdData[0];
```

Comment:
Have you considered `formData.householdData.length` === 0` to make it more explicit?

Reply:
It seems like `formData.householdData.length >= 1` would be better if we want to be explicit.

Reply:
Oops. You are right. 🤦‍♂️